core/os/input_event.cpp

```diff
@@ -610,14 +640,29 @@ void InputEventJoypadMotion::set_axis_value(float p_value) {
 
 	axis_value = p_value;
 }
+
 float InputEventJoypadMotion::get_axis_value() const {
 
 	return axis_value;
 }
 
+void InputEventJoypadMotion::set_deadzone(float p_deadzone) {
```

Comment:
I am not convinced about this, setting deadzone here is wrong.  I would use the axis_value as deadzone in the action and forget this.